test-suite/Makefile.am

```diff
@@ -140,9 +141,25 @@ VirtualDeleteOperator_SOURCES = VirtualDeleteOperator.cc stub_libmem.cc $(DEBUG_
 ##$(TARGLIB): $(LIBOBJS)
 ##	$(AR_R) $(TARGLIB) $(LIBOBJS)
 
-squid-conf-tests: $(top_builddir)/src/squid.conf.default $(srcdir)/squidconf/*
+tls-test-certs: $(srcdir)/mk-tls-test-certs.sh
```

Comment:
Please rename `tls-test-certs` to something more telling and less ambiguous, like `generate-certificates`. I do not think we should add "test" to every name inside a `test-suite` directory, but `generate-test-certificates` would also work. As you know from another PR review, I do not think these certificates should be called "TLS certificates", but I am not going to fight your obsession with those three letters if you insist on `generate-tls-certificates`.

Reply:
Done. Though this was meant as "Certificates - for the tests the TLS features". In contrast to the abundant SSL tests already existing.

Reply:
In a general context like this one, folks normally use TLS and SSL interchangeably and, hence, will not be able to infer what you meant.